pkgs/development/python-modules/nose-randomly/default.nix

```
@@ -16,9 +17,10 @@ buildPythonPackage rec {
 
   checkInputs = [ numpy nose ];
 
-  checkPhase = ''
-    nosetests
-  '';
+  checkPhase = if stdenv.isDarwin then ''
+    # Work around "OSError: AF_UNIX path too long"
+    TMPDIR="/tmp" nosetests
+  '' else "nosetests";
```

```suggestion
  '' else ''
    nosetests
  '';
```
That should avoid the rebuilds on linux

---

TIL! Do you want me to squash back to one commit?

---

I just squash merged.